src/main/java/org/intellij/sdk/codesync/codeSyncSetup/CodeSyncSetupAction.java

```diff
@@ -95,6 +98,37 @@ public void actionPerformed(@NotNull AnActionEvent e) {
                     // Show Pricing alert dialog.
                     PricingAlerts.setPlanLimitReached(project);
                 } else {
+                    // Get access token
+                    String accessToken;
+                    try {
+                        UserAccount userAccount = new UserAccount();
+                        accessToken = userAccount.getActiveAccessToken();
+                    } catch (SQLiteDBConnectionError err) {
+                        accessToken = null;
+                    }
+//                    // User already has access token, no need to proceed.
+//                    if (accessToken != null) {
+//                        try {
+//                            return validateAccessToken(accessToken);
+//                        } catch (InvalidAccessTokenError invalidAccessTokenError) {
+//                            // no action needed, user will be prompted for authenticated later by default.
+//                        }
+//                    }
+                    // Hit /v1/init endpoint
+                    JSONResponse jsonResponse;
+                    try {
+                        jsonResponse = ClientUtils.sendPost(API_INIT, accessToken);
```

Comment:
@saadberry Aren't we doing this already in the code somewhere? The code that "Connects a Repo". We need to add error handling on the existing API call